uniswap-v2-core/contracts/UniswapV2Factory.sol

```diff
@@ -2,18 +2,28 @@ pragma solidity =0.5.16;
 
 import './interfaces/IUniswapV2Factory.sol';
 import './UniswapV2Pair.sol';
+import './libraries/Ownable.sol';
+
+contract UniswapV2Factory is IUniswapV2Factory, Ownable {
```

Comment:
Weird that uniswap didn't use this?
Did they need to save so much on the size of the contract?

Reply:
I think it was for optics, `Ownable` goes against crypto ethos, even though it doesn't grant custodial powers